store/bigcache.go

```diff
@@ -59,13 +59,24 @@ func (s *BigcacheStore) GetWithTTL(key interface{}) (interface{}, time.Duration,
 	return item, 0, err
 }
 
-// Set defines data in Redis for given key identifier
+// Set defines data in Bigcache for given key identifier
 func (s *BigcacheStore) Set(key interface{}, value interface{}, options *Options) error {
 	if options == nil {
 		options = s.options
 	}
 
-	err := s.client.Set(key.(string), value.([]byte))
+	var val []byte
+	switch v := value.(type) {
+	case string:
+		val = []byte(v)
+	case []byte:
+		val = v
+	default:
+		// this convert may cause panic
+		val = value.([]byte)
```

Comment:
I think we should return an error in this fallback case, like it's done here: https://github.com/eko/gocache/blob/master/store/freecache.go#L99

What do you think?

Also, I think we could just create a `convertToBytes` function that could be used by stores that needs this logic in order to have the same logic everywhere. I can do it later if you prefer.

Reply:
> I think we should return an error in this fallback case, like it's done here

OK, I can do it.

> Also, I think we could just create a convertToBytes function that could be used by stores that needs this logic in order to have the same logic everywhere. I can do it later if you prefer.

That's great!